package/kedro_viz/server.py

```diff
@@ -527,6 +549,19 @@ def _add_parameter_data_to_node(dataset_namespace, task_id):
         _JSON_NODES[task_id]["parameters"][parameter_name] = parameter_value
 
 
+def _extract_dataset_modular_pipelines(dataset_full_name):
```

Comment:
Could we reduce the nested calling of functions by having a function: `_get_namespace(dataset_full_name)` and then simply call at line 499:

`_expand_namespaces(_get_namespace(dataset_full_name))`?

This looks a tad easier to understand for me since each function does only one thing, rather than one building on the other.